crates/autopilot/src/arguments.rs

```diff
@@ -407,6 +417,22 @@ impl FromStr for FeePolicyKind {
                     max_volume_factor,
                 })
             }
+            "price-improvement" => {
```

Comment:
I believe we currently tend to prefer `camelCase` over all so it makes sense to also use it here IMO.